elixir/apps/domain/lib/domain/auth/adapters/okta/api_client.ex

```diff
@@ -100,6 +100,7 @@ defmodule Domain.Auth.Adapters.Okta.APIClient do
     end
   end
 
+  # TODO: Need to catch 401/403 specifically when error message is in header
```

Comment:
TODO left here

Reply:
This was intentionally left.  I wanted to make the Okta updates a ticket on their own for a couple of reasons... 1) I wasn't sure if this would open up a little bit of a can of worms and 2) if we're changing over to service accounts anyway, this may not be needed since service accounts use a different auth method, which may not return the same type of response.